CHANGELOG.md

```diff
@@ -30,6 +30,7 @@ https://github.com/nwnxee/unified/compare/build8193.36.12...HEAD
 - Player: added bChatWindow parameter to FloatingTextStringOnCreature() 
 - Damage: added iSpellId to the NWNX_Damage_DamageEventData struct.
 - Docker: Add dotnet-runtime-8.0, dotnet-apphost-pack-8.0 packages.
+- Item: Added parameter `bUdateCreatureAppearance` to SetItemAppearance() to update the appearance of the item's possessor.
```

Comment:
Is that supposed to be "bUpdateCreatureAppearance"?

Reply:
Oops